lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -526,13 +526,13 @@ public function notLike($x, $y)
      * Creates a CONCAT() function expression with the given arguments.
      *
      * @param mixed $x First argument to be used in CONCAT() function.
-     * @param mixed $y Second argument to be used in CONCAT() function.
+     * @param mixed $y,... Other arguments to be used in CONCAT() function.
      *
      * @return Expr\Func
      */
     public function concat($x, $y)
```

Comment:
@zeroedin-bill well I don't know if this is strictly a BC.

I mean, to stay on the safe side the `$y` parameter should be kept, but at runtime is a BC only to add a parameters (w/o defaults), since PHP will ignore extra parameters by default.

You'll have a BC by overriding the method in sub classes, but since this class is part of a Builder pattern is meant to be final anyway.

Am I correct?